save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/ExecutionView.kt

```diff
@@ -422,13 +422,18 @@ class ExecutionView : AbstractView<ExecutionProps, ExecutionState>(false) {
                 }
             }
             getPageCount = { pageSize ->
-                val count: Int = post(
-                    url = "$apiUrl/test-executions?executionId=${props.executionId}&page=1&size=$pageSize&checkDebugInfo=true",
-                    headers = Headers().apply {
-                        set("Accept", "application/json")
-                        set("Content-Type", "application/json")
+                val status = filters.status?.let {
+                    "&status=${filters.status}"
+                } ?: ""
+                val testSuite = filters.testSuite?.let {
+                    "&testSuite=${filters.testSuite}"
+                } ?: ""
+
```

Comment:
Maybe something like 
```kotlin
val filtersQueryString = buildString {
   filters.status?.let  { append(...) }
}
```
to make it easier to incorporate new filters should they be needed in the future?